synapse/lib/cell.py

```diff
@@ -363,6 +359,14 @@ def setCellUser(self, iden):
         mesg = 'getRoleInfo denied for non-admin and non-member'
         raise s_exc.AuthDeny(mesg=mesg)
 
+    @adminapi
```

Comment:
These were added to facilitate optic's need for checking perms and adding users to a local core or a core proxy